person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] I would advise you to use a random number for the @id, it will not be needed to include it in the arguments either.